.github/workflows/pr.yml

```diff
@@ -54,8 +64,8 @@ jobs:
     - name: Get dependencies
       run:  go mod download
         
-    - name: Lint code
-      run: GOGC=300 make lint-code
+  #  - name: Lint code
+  #    run: GOGC=300 make lint-code
```

Comment:
for lint error,  we should fix it in code.  please don't comment out this.

Reply:
I need some help with that. I'm getting the following error when this is enabled: 
"Run GOGC=300 make lint-code
GO111MODULE=on   golangci-lint -v run --timeout 3m ./...
panic: load embedded ruleguard rules: rules/rules.go:13: can't load fmt"
Any idea here what might be wrong?

Reply:
I previous met similar issue when upgrade csi devkit golintci-lint versions(https://github.com/dell/csi-baremetal/issues/1016).  it is a tools issues.  
I think you'd better do a quick google search to this errmsg check it is a tools issue and which version fix it.  also do we need to upgrade this tool to the latest, is it necessary ?